### DISCUSSION TITLE
PDF builds for version 0.12

### DISCUSSION THREAD

**Discussion by nsushkin** [winitzki/sofp#44] [General]
Hi, Serge,

I don't see any PDFs under assets in version 0.12.0. The most recent PDFs seems to be in https://github.com/winitzki/sofp/releases/tag/v0.11.2

Are your builds failing or do you need to do something special to post PDF assets for the latest revision of your book?

Thanks!

Nick

**Comment by winitzki** [comment]
Hi @nsushkin - Sorry for the late answer, I don't get notified about discussions for some reason.

The PDF assets are published by Git in a special place. I documented this in the README: 

The "Releases" section of this repository contains (under "Assets") the full uncorrected draft (sofp.pdf) and the finished and proof-read part of the draft (sofp-draft.pdf). The latest release is GitHub release and represents the most recent, relatively stable version of the text.

The PDF version of the most recent master branch of the book can be downloaded by following the badge link to the PDF builds, clicking on the latest PDF build, and then clicking on "PDF files and LaTeX logs".

**Comment by dmanikhine** [comment]
Unfortunately link "PDF files and LaTeX logs" is expired.

**Reply by winitzki** [reply]
Yes, thank you.

These links expire after 3 months. I will rebuild the master branch to refresh them.
